src/Connection.php

```diff
@@ -169,6 +174,11 @@ protected function createConnection($dsn, array $config, array $options): Client
             $driverOptions = $config['driver_options'];
         }
 
+        $driverOptions['driver'] = [
+            'name' => 'laravel-mongodb',
+            'version' => self::getVersion(),
```

Comment:
Is this affects performance in any case?

Reply:
Not that I'm aware of. This is composer functionality and it is called once per request if a connection is established. Composer itself optimises this to a significant degree, so I wouldn't consider it performance critical. We've been using this both in the PHP library as well as the Doctrine MongoDB ODM without any reports of performance degradation.